diff --git a/dist/node/vite/index.js b/dist/node/vite/index.js
index 403d00df62ac83beb64db23d4ff3fbd3c9457ee7..e1588b304144142d79fada86e1845db450339d24 100644
--- a/dist/node/vite/index.js
+++ b/dist/node/vite/index.js
@@ -30,7 +30,6 @@ import { pluginSuppressRollupWarning } from './plugins/build/pluginSuppressRollu
 import { pluginDistFileNames } from './plugins/build/pluginDistFileNames.js';
 import { pluginProdBuildEntry } from './plugins/build/pluginProdBuildEntry.js';
 import { pluginBuildConfig } from './plugins/build/pluginBuildConfig.js';
-import { pluginModuleBanner } from './plugins/build/pluginModuleBanner.js';
 import { pluginReplaceConstantsNonRunnableDev } from './plugins/non-runnable-dev/pluginReplaceConstantsNonRunnableDev.js';
 import { isVikeCliOrApi } from '../../shared-server-node/api-context.js';
 import { pluginViteConfigVikeExtensions } from './plugins/pluginViteConfigVikeExtensions.js';
@@ -77,7 +76,6 @@ function pluginBuild() {
         ...pluginDistPackageJsonFile(),
         ...pluginSuppressRollupWarning(),
         ...pluginDistFileNames(),
-        ...pluginModuleBanner(),
     ];
 }
 function pluginNonRunnabeDev() {
