diff --git a/dist/hooks/useData.d.ts b/dist/hooks/useData.d.ts
index ccf9484ed04ac8b46622160f2496172dc5a9063a..24056240e094719be1a3b347ea68a049c338877b 100644
--- a/dist/hooks/useData.d.ts
+++ b/dist/hooks/useData.d.ts
@@ -1,9 +1,11 @@
+import { SetStoreFunction } from 'solid-js/store';
+
 /** Access `pageContext.data` from any SolidJS component
  *
  * See
  * - https://vike.dev/data
  * - https://vike.dev/pageContext-anywhere
  */
-declare function useData<Data>(): Data;
+declare function useData<Data>(): [Data, SetStoreFunction<Data>];
 
 export { useData };
diff --git a/dist/hooks/useData.js b/dist/hooks/useData.js
index 1ac40d76b3b311c5b6871fe4ba23cfb906c8930c..3367e7623054e9d4961399e3a62e47eea807a691 100644
--- a/dist/hooks/useData.js
+++ b/dist/hooks/useData.js
@@ -17,7 +17,7 @@ function useData() {
   createEffect(() => {
     setData(pageContext?.data);
   });
-  return data;
+  return [data, setData];
 }
 
 export { useData };
