diff --git a/dist/node/vite/index.js b/dist/node/vite/index.js
index 983684a5e65b08db7719c9e2a47624c1f66f4051..c9b14d0f33992105b9d53d7ce7fd028dd92f6407 100644
--- a/dist/node/vite/index.js
+++ b/dist/node/vite/index.js
@@ -33,7 +33,6 @@ import { pluginSuppressRollupWarning } from './plugins/build/pluginSuppressRollu
 import { pluginDistFileNames } from './plugins/build/pluginDistFileNames.js';
 import { pluginProdBuildEntry } from './plugins/build/pluginProdBuildEntry.js';
 import { pluginBuildConfig } from './plugins/build/pluginBuildConfig.js';
-import { pluginModuleBanner } from './plugins/build/pluginModuleBanner.js';
 import { pluginReplaceConstantsNonRunnableDev } from './plugins/non-runnable-dev/pluginReplaceConstantsNonRunnableDev.js';
 import { isVikeCliOrApi } from '../../shared-server-node/api-context.js';
 import { pluginViteConfigVikeExtensions } from './plugins/pluginViteConfigVikeExtensions.js';
@@ -80,7 +79,6 @@ function pluginBuild() {
         ...pluginDistPackageJsonFile(),
         ...pluginSuppressRollupWarning(),
         ...pluginDistFileNames(),
-        ...pluginModuleBanner(),
     ];
 }
 function pluginNonRunnabeDev() {
