diff --git a/dist/node/vite/index.js b/dist/node/vite/index.js
index e669072b061680424cbb50888f311d700b9b3036..8b619b3e7a95f793fd7b10a330f552f5a70a5220 100644
--- a/dist/node/vite/index.js
+++ b/dist/node/vite/index.js
@@ -32,7 +32,6 @@ import { pluginSuppressRollupWarning } from './plugins/build/pluginSuppressRollu
 import { pluginDistFileNames } from './plugins/build/pluginDistFileNames.js';
 import { pluginProdBuildEntry } from './plugins/build/pluginProdBuildEntry.js';
 import { pluginBuildConfig } from './plugins/build/pluginBuildConfig.js';
-import { pluginModuleBanner } from './plugins/build/pluginModuleBanner.js';
 import { pluginReplaceConstantsNonRunnableDev } from './plugins/non-runnable-dev/pluginReplaceConstantsNonRunnableDev.js';
 import { isVikeCliOrApi } from '../../shared-server-node/api-context.js';
 import { pluginViteConfigVikeExtensions } from './plugins/pluginViteConfigVikeExtensions.js';
@@ -88,7 +87,6 @@ function pluginBuild() {
         ...pluginDistPackageJsonFile(),
         ...pluginSuppressRollupWarning(),
         ...pluginDistFileNames(),
-        ...pluginModuleBanner(),
     ];
 }
 function pluginNonRunnabeDev() {
